### PR TITLE
add example showing parameterised label values

### PR DESCRIPTION
**Pull request by jim-minter**
Documentation for https://github.com/openshift/origin/pull/16605 - to be merged after the Origin PR merges.

### CODE REVIEW

**Comment by jim-minter** [comment]
@bparees ptal

File: dev_guide/templates.adoc [hunk 1]
```diff
@@ -444,8 +444,11 @@ apiVersion: "v1"
 ...
 labels:
   template: "cakephp-mysql-example" <1>
+  app: "${NAME}" <2>
```
Comment (bparees) [inline dev_guide/templates.adoc]: assuming you agree we should support key substitution, update this to illustrate that too and then lgtm.

Comment (jim-minter) [reply dev_guide/templates.adoc]: done

**Comment by jim-minter** [comment]
@bmcelvee this PR might have got forgotten - ptal?

**Comment by bmcelvee** [comment]
Thanks @jim-minter, on it! I'm sorry I didn't see it sooner!

**Comment by bmcelvee** [comment]
LGTM

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../dev_guide/templates.adoc#dev-guide-templates[Templates]
|Added example demonstrating parametrized label values in the xref:../dev_guide/templates.adoc#writing-labels[Labels] section.
%

**Pull request merged**